docs/section5/Rule5-43.md

```diff
@@ -0,0 +1,44 @@
+
+# Envelope - Rule 5-43  
+
+**Rule ID:** 5-43  
+**Rule Description:** The  proposed roof surfaces shall be modeled using the same solar reflectance as in the user model.  
+**Rule Assertion:** P-RMR SurfaceOpticalProperties:absorptance_solar_exterior = U-RMR SurfaceOpticalProperties:absorptance_solar_exterior  
+**Appendix G Section:** Section G3.1-1(a) Building Envelope Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. get_opaque_surface_type()
+  2. get_surface_conditioning_category()
+  3. match_data_element()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for P_RMR: `scc_dictionary_p = get_surface_conditioning_category(P_RMR)`
+
+- For each building segment in the Proposed model: `for building_segment_p in P_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_p in building_segment_p.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_p in thermal_block_p.zones:`
+
+      - For each surface in zone: `for surface_p in zone_p.surfaces;`
+
+        - Check if surface is roof and is regulated, get matching surface in U_RMR: `if ( get_opaque_surface_type(surface_p.id) == "ROOF" ) AND ( scc_dictionary_p[surface_p.id] != "UNREGULATED" ): surface_u = match_data_element(U_RMR, Surfaces, surface_p.id)`
+
```

Comment:
remove the check that this is a regulated surface.

Reply:
Removed.